### PR TITLE
[Frontend] Introduces the Chinese language version of the Frontend

### PR DESCRIPTION
**Pull request by gzhu2**
## What is the purpose of the change

汉化画面参与前端开发

## Brief change log

可以为国内潜在用户快速展示原型

### CODE REVIEW

**Comment by jiangxin369** [comment]
Hi @gzhu2, thanks for your pull request, could you please fulfill the necessary information like `Brief change log` which helps review?

**Comment by gzhu2** [comment]
Hi

从汉化画面参与前端开发
可以为国内潜在用户快速展示原型为目的.

Pull request已更新
请查阅.

/zhu

________________________________

File: ai_flow/frontend/src/locales/lang/zh-CN.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  momentName: 'zh-cn',
+  momentLocale: momentZH
+}
+
+export default {
+  message: '-',
+
+  'layouts.usermenu.dialog.title': '信息',
+  'layouts.usermenu.dialog.content': '您确定要退出吗?',
+  'layouts.userLayout.title': 'AIFlow是一个开源框架，将大数据和人工智能联系起来.',
```
Comment (SteNicholas) [inline ai_flow/frontend/src/locales/lang/zh-CN.js:44]:
```suggestion
  'layouts.userLayout.title': 'AIFlow 是一个将大数据和人工智能连接的开源工作流框架。',
```

File: ai_flow/frontend/src/locales/lang/zh-CN.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  ...components,
+  ...menu,
+  ...setting,
+  ...user,
+  ...project,
```
Comment (SteNicholas) [inline ai_flow/frontend/src/locales/lang/zh-CN.js:49]: Ditto.

**Review by SteNicholas — CHANGES_REQUESTED** [review]

**Comment by SteNicholas** [comment]
@gzhu2 , any update for this pull request?

**Pull request closed without merging**